main.py

```
@@ -6,37 +6,26 @@
 # TODO also report the day
 # TODO would be nice to move to a subdirectory
 
-import re
-import sys
+import flask
```

Pylint (3.3.7)

[error] Unable to import 'flask'

(E0401)